tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -47,7 +47,11 @@ sub run {
     $variables{HANA_ACCOUNT} = get_required_var('QESAPDEPLOY_HANA_ACCOUNT');
     $variables{HANA_CONTAINER} = get_required_var('QESAPDEPLOY_HANA_CONTAINER');
     if (get_var('QESAPDEPLOY_HANA_TOKEN')) {
-        $variables{HANA_TOKEN} = get_required_var('QESAPDEPLOY_HANA_TOKEN');
+        $variables{HANA_TOKEN} = qesap_az_create_sas_token(storage => get_required_var('QESAPDEPLOY_HANA_ACCOUNT'),
+            container => get_required_var('QESAPDEPLOY_HANA_CONTAINER'),
+            keyname => get_required_var('QESAPDEPLOY_HANA_KEYNAME'),
```

Comment:
This is a new setting

Reply:
If it's a new setting and its contents are sensitive, rename it as `_SECRET_QESAPDEPLOY_HANA_KEYNAME` so it's not printed in `vars.json` after test finish. If content is not sensitive, keep the name you're proposing.